kong/tools/batch_queue.lua

```diff
@@ -258,6 +266,17 @@ function Queue.new(process, opts)
 end
 
 
+-------------------------------------------------------------------------------
+-- Log a message that includes the name of the queue for identification purposes
+-- @param self Queue
+-- @param level: log level
+-- @param formatstring: format string, will get the queue name and ": " prepended
+-- @param ...: formatter arguments
+function Queue:log(level, formatstring, ...)
+  return ngx.log(level, string.format(self.name .. ": " .. formatstring, unpack({...})))
```

Comment:
```suggestion
  return ngx.log(level, string.format("batch-queue [" .. self.name .. "]: " .. formatstring, unpack({...})))
```

Something in the log line that indicates that this log is coming from this file or module would help.

Reply:
The filename (`batch_queue.lua`) is included by default in our loglines already, I don't think this particular change is necessary. It will look like:

> ... [lua] batch_queue.lua:276: <name>: <msg>

If we were going to go that direction, I would use the same format for tagging loglines that we use in other places like timer-ng or cluster events, which are preceded by a `[timer-ng]` and `[cluster_events]`:

```suggestion
  return ngx.log(level, string.format("[batch-queue] self.name .. ": " .. formatstring, unpack({...})))
```

Reply:
Given that this is a band-aid only, can we leave cleaning up the log messages and the documentation to the queue rework fix?